src/h_assets/cached_file.py

```diff
@@ -0,0 +1,75 @@
+import configparser
```

Comment:
Should this be `_cached_file.py`? My original intent was that `Environment` and `assets_view` would be the only public API of this package, imported at the root. I don't know what the current conventions are for Python library projects and indicating the privacy/API surface of a package. I have seen the use of underscore-prefixed modules in recent h and lms code though.

Reply:
I've done this now. Previously I didn't import the symbols into the main `__init__.py` so anyone doing `from h_assets ...` wouldn't see it.